docker/Dockerfile

```diff
@@ -83,9 +83,9 @@ COPY --from=pre-builder /gems/ /gems/
 COPY --from=pre-builder /app /app
 
 # Remove unnecessary files
-RUN rm -rf /gems/ruby/3.0.2/cache/*.gem \
-  && find /gems/ruby/3.0.2/gems/ -name "*.c" -delete \
-  && find /gems/ruby/3.0.2/gems/ -name "*.o" -delete
+RUN rm -rf /gems/ruby/3.0.0/cache/*.gem \
+  && find /gems/ruby/3.0.0/gems/ -name "*.c" -delete \
+  && find /gems/ruby/3.0.0/gems/ -name "*.o" -delete
```

Comment:
```suggestion
  && find /gems/ruby/3.0.0/gems/ \( -name "*.c" -o -name "*.o" \) -delete
```

Reply:
Thanks! Was about to submit the same fix 👍 Just suggesting a small change to combine the two `find` commands into one.

Reply:
Nice catch 😃